### DISCUSSION TITLE
Credentials on Laravel vapor

### DISCUSSION THREAD

**Discussion by gera919** [freekmurze/laravel-analytics#478] [Q&A]
Does anyone knows how to store the google credentials.json file or data when using serverless environments like Laravel Vapor? 

Of course tracking the file on the repo is not an option for security purposes. 

Laravel Vapor has a restriction limit of 2,000 max characteres so moving the credential values to the .env file is not an option either.

Already reached the Vapor support team looking for a solution but it doesn't looks promising.

I'm getting frustrated since I cannot find a proper solution for this and cannot go live with the analytics feature because of this.

Am I missing something?

Thanks in advance!

**Comment by alejandrozepeda** [comment]
You can add the private key as "Secrets" in the vapor dashboard, so is injected into the environment variables when the application is deployed
